lib/purchases_flutter.dart

```diff
@@ -351,69 +347,69 @@ class Purchases {
   /// [newAppUserID] The new appUserID that should be linked to the currently
   /// identified appUserID.
   @Deprecated('Use logIn instead.')
-  static Future<PurchaserInfo> createAlias(String newAppUserID) async {
+  static Future<CustomerInfo> createAlias(String newAppUserID) async {
     final result = await _channel
         .invokeMethod('createAlias', {'newAppUserID': newAppUserID});
-    return PurchaserInfo.fromJson(Map<String, dynamic>.from(result));
+    return CustomerInfo.fromJson(Map<String, dynamic>.from(result));
   }
 
   /// This function will logIn the current user with an appUserID.
   /// Typically this would be used after logging in a user to identify them without
   /// calling configure
   ///
   /// Returns a [LogInResult] object, or throws a [PlatformException] if there
-  /// was a problem restoring transactions. LogInResult holds a PurchaserInfo object
+  /// was a problem restoring transactions. LogInResult holds a [CustomerInfo] object
   /// and a bool that can be used to know if a user has just been created for the first time.
   ///
   /// [newAppUserID] The appUserID that should be linked to the currently user
   static Future<LogInResult> logIn(String appUserID) async {
     final result =
         await _channel.invokeMethod('logIn', {'appUserID': appUserID});
-    final purchaserInfo = PurchaserInfo.fromJson(
-      Map<String, dynamic>.from(result['purchaserInfo']),
+    final customerInfo = CustomerInfo.fromJson(
+      Map<String, dynamic>.from(result['customerInfo']),
```

Comment:
**This** is a delicate one - purchasesHybridCommon needs to also send `customerInfo`. Currently it's sending `purchaserInfo` (at least on iOS, up to [this PR](https://github.com/RevenueCat/purchases-hybrid-common/pull/129)). The PR after that one will modify this string

Reply:
Since it's the only place where we depend on a value sent by hybrid common, perhaps we can have only this change on a separate PR?

Reply:
You should be able to test this out with MagicWeather, there's a login option

Reply:
I reverted the change and added todos. I think it makes sense as a separate PR